src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -6201,6 +6203,9 @@ public static String removeStartIgnoreCase(final String str, final String remove
         return str;
     }
 
+    // Padding
```

Comment:
Let's not use these kind of separator. I usually prefer to keep the methods sorted in AB order so its simple and easy to find methods.

Reply:
@garydgregory 
Hi.
I didn't ADD this seperator, but MOVE it to a better place.
you can see it append in a weird place in the original codes.
If you think it is better to delete it, then I'll be glad to delete it.
I don't like ie either :)

Reply:
@garydgregory resolved as deleted.